checkout-ui-custom/src/utils/phoneNumberFormat.js

```diff
@@ -0,0 +1,75 @@
+import { GOOGLE_LIBPHONELIBRARY_URL } from "./const";
+
+const usePhoneNumberFormatting = () => {
+  let phoneUtil = null;
+
+
+  const loadPhoneNumberUtil = () => {
```

Comment:
@harlyon you don't have to load it like this - you can use the node module in Checkout, which will be more reliable.

Reply:
@simmbiote not sure how this works, can we sync up on this